components/compliance-service/api/tests/run.rb

```diff
@@ -110,6 +111,10 @@ def assert_equal_json_content(expected, actual)
   def assert_equal_json_sorted(expected, actual)
     assert_equal JSON.parse(expected).deep_sort, JSON.parse(actual).deep_sort
   end
+
+  def assert_same_elements(expected, actual)
+    assert_equal(expected.to_set, actual.to_set)
+  end
```

Comment:
Adding `DISTINCT` seemed to have the side effect of making postgres alphabetize the values(?), so rather than change the expected values for the tests or add `.sort` calls all over the place, I added this test helper that tests that the same elements were returned without caring about the order.